lightly/models/utils.py

```diff
@@ -678,20 +715,24 @@ def random_block_mask_image(
         num_masks:
             Number of patches to mask.
         min_num_masks_per_block:
-            Minimum number of patches to mask per block.
+            Minimum number of patches to mask per block. Default is 4.
         max_num_masks_per_block:
             Maximum number of patches to mask per block.
         min_block_aspect_ratio:
-            Minimum aspect ratio (height/width) of a masked block.
+            Minimum aspect ratio (height/width) of a masked block. Default is 0.3.
         max_block_aspect_ratio:
             Maximum aspect ratio (height/width) of a masked block.
         max_attempts_per_block:
-            Maximum number of attempts to find a valid block mask.
+            Maximum number of attempts to find a valid block mask. Default is 10.
         device:
             Device on which to create the mask.
     Returns:
         A boolean tensor with shape (height, width) where each entry is True if the
         patch should be masked and False otherwise.
+
+    Raises:
+        ValueError: If 'max_num_masks_per_block' is less than 'min_num_masks_per_block' or
+                    if 'max_block_aspect_ratio' is less than 'min_block_aspect_ratio'    
```

Comment:
```suggestion
    Raises:
        ValueError: If 'max_num_masks_per_block' is less than 'min_num_masks_per_block' or
            if 'max_block_aspect_ratio' is less than 'min_block_aspect_ratio'    
```

We usually don't align indentation with text from previous line. Instead, we just indent one tab more.

Reply:
@guarin done